components/social-media-posts/RedditPostComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import SocialMediaButtonsContainer from 'components/SocialMediaButtonsContainer'
+import Headline from "components/Headline"
+import AvatarComponent from "components/AvatarComponent"
+import { jsx } from '@emotion/react'
+
+interface Props {
+  avatarAlt: string;
+  avatarChildren: string;
+  avatarSrc: string;
+  children: React.ReactNode;
+  className: string;
+}
+  
+const RedditPost = ({
+  avatarAlt,
+  avatarChildren,
+  avatarSrc,
+  className,
+  children,
+}: Props): JSX.Element=> (
+  <>
+    <div className={`${className}`}>
+      <AvatarComponent
+        alt={avatarAlt}
+        children={avatarChildren}
```

Comment:
as a general rule of thumb, we probably should never be providing children as props and instead provide them in the following format
```
<component>
  {children}
</component>